### ISSUE TITLE
delete tmp files for DMs

### ISSUE THREAD

**Issue by julianseeger** [Qabel/qabel-desktop#321]
Tmp files (the actual sqlite files) of DMs cannot be deleted currently, because the DM is helt open all the time (until it is legacy). The tmp files are only deleted on GC which holds the number of files about constant while qabel is running but leaves tmp files on exit.

We need a strategy to clean them. Maybe a constant qabel tmp dir that is clean on start or something like that.

**Comment by audax** [comment]
Is this already adressed with the changes in the core? @julianseeger

**Comment by julianseeger** [comment]
Nope. We have a pretty good clean up one streams are closed or DMs are GC'ed but a dedicated qabel tmp dir that we can wipe as we feel like would still be an improvement ... a final solution
